### DISCUSSION TITLE
Currently does not seem to support attacking node

### DISCUSSION THREAD

**Discussion by KingsonKai** [chaos-mesh/chaos-mesh#3952] [Q&A]
It seems that attacking node is not currently supported, will it be supported in the future?

**Comment by STRRL** [comment]
Please take a look on PhysicalMachineChaos and chaosd:

- https://chaos-mesh.org/docs/simulate-physical-machine-chaos/
- https://github.com/chaos-mesh/chaosd/

**Reply by KingsonKai** [reply]
I have learned about this, but this method requires manual installation of chaosd in advance, and it needs to be installed on each machine. Is there a similar method of attacking pods based on daemonsets, which can attack cluster nodes with one deployment?

**Reply by STRRL** [reply]
Chaos Mesh did not provide install chaosd on each machine automatically right now. We also received a similar demand/feature request about that, but we do not have a proposal yet. 

Feel free to contribute :)
